Userland/DevTools/SQLStudio/MainWidget.cpp

```diff
@@ -138,7 +139,16 @@ MainWidget::MainWidget()
     });
 
     m_run_script_action = GUI::Action::create("Run script", { Mod_Alt, Key_F9 }, Gfx::Bitmap::try_load_from_file("/res/icons/16x16/go-forward.png").release_value_but_fixme_should_propagate_errors(), [&](auto&) {
-        TODO();
+        auto editor = dynamic_cast<ScriptEditor*>(m_tab_widget->active_widget());
+        if (!editor)
+            return;
+        // TODO select the database to use in UI.
+        auto connection_id = m_sql_client->connect("test");
+        auto sql_statements = editor->document().text().split(';');
```

Comment:
What if there is a semicolon inside a string, or a comment? This won't work. can the SQL client not deal with multiple statements at once?

Reply:
True.. I changed the implementation, now I've kind of copied what was in the sql.cpp utility. Still thinking perhaps I should factor out the code.. into the SQLClient?